static/css/components/wmd.less

```diff
@@ -10,11 +10,14 @@
   background-color: transparent;
 }
 
+fieldset.major .wmd-button-bar {
```

Comment:
This adds some additional CSS

I think in this file it makes sense to wrap the rule with 
```
/* stylelint-disable no-descending-specificity */ 
a:hover...
/* stylelint-enable no-descending-specificity */ 
```
and leave this file unchanged.